docs/integration_resources/software/nuke.md

```diff
@@ -14,59 +14,107 @@ nshift.show()
 
 ## Shift installation in Nuke
 
-To install Shift and open it through a *menu.py* entry within Nuke, it is required to set up Shift in the user *init.py* and **menu.py*.py* Python files from Nuke. Shift can be added to the user *init.py* and *menu.py* files from Nuke's preference folder (*<home directory>/.nuke*) or to a custom *init.py* and *menu.py* files inside a custom folder structure. If the directory where these files are stored is a path sourced by Nuke, they will be automatically executed on startup.
+To install Shift and open it through a menu entry within Nuke, it is required to set up Shift in the environment or in the user *init.py* Python files from Nuke. Shift can be added to the user *init.py* file from Nuke's preference folder (*<home directory>/.nuke*) or to a custom *init.py* file inside a custom folder structure. If the directory where these files are stored is a path sourced by Nuke, they will be automatically executed on startup.
 
-### Init File
+### Init File/Environment requirements
 
-In the *init.py* file, it is necessary to configure the requirements for Shift to work in Nuke batch and interactive modes. To achieve this, it is only required to add the Shift installation path to the **PATH** environment variable.
+You can add these requirements to your environment before opening Nuke:
 
+<pre><code style="white-space: pre; margin: 20px 0; padding: 10px; box-sizing: border-box;">
+PATH=&#37;PATH&#37;&semi;"&ltpath_to_the_shift_installation_folder&gt"
+PATH=&#37;PATH&#37;&semi;"&ltpath_to_the_shift_installation_folder&gt/shift/thirdparty/python/Lib/site-packages"
+NUKE_PATH="&ltpath_to_the_shift_installation_folder&gt/shift/plugins/nuke/startup"&semi;&#37;NUKE_PATH&#37;
 
-<pre><code style="white-space: pre; margin: 20px 0; padding: 10px; box-sizing: border-box;">try:
+</code></pre>
+
+Or you can add them to the *init.py* file to make Shift to work in Nuke batch and interactive modes. To achieve this, it is only required to add the Shift installation path to the **PATH** environment variable.

```

Comment:
This is not exactly correct as appending to sys.path does not change the environment variable PATH but only the python resources.
```suggestion
Or you can add them to the *init.py* file to source Shift in Nuke batch and interactive modes. To achieve this, it is only required to add the Shift installation path to the Python `sys.path`.

```